react-function-components/src/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton(props) {
+  return <button>Click Me!</button>;
+}
+
+const container = document.getElementById('root');
+
+const root = ReactDOM.createRoot(container);
+
+root.render(CustomButton());
```

Comment:
With components we never directly call them. Instead we write them as JSX:
```JSX
root.render(<CustomButton />);
```

Reply:
ah good to know! addressed. definitely missed that.